reference/library/src/morpheus-master/sass/base/_bootstrap-defaults.scss

```diff
@@ -145,6 +145,25 @@ $icon-font-path: if($bootstrap-sass-asset-helper, "bootstrap/", "../" + $skin-na
 //** Border color for table and cell borders.
 // $table-border-color:            #ddd
 
+// SAK-32144 override the nowrap behavior of table-responsive
+.table-responsive {
+  @media screen and (max-width: 480px) {
```

Comment:
Or @RebeccaOxfordUni pointed out to me that we have some sizes defined in `_defaults.scss` and if we're not going to use the bootstrap sizes should we define the size in there and refer to it?